hedyorg/hedy
# 🪲 fix custom adventures' alignment

Fixes #5296 
Fixes #5293

**Description**
In this PR, I attempted to split the teachers' adventures into two columns optimally. This is done by splitting the tags rather than the text, as we currently do. Furthermore, I've set a maximum height on the code blocks to equal the editor's height. This is to avoid very long code blocks which would most likely smaller than other information within the adventure content.

**How to test?**
- create long adventures
- add them to your class
- preview the class and whether the adventure is split intow 2 columns, the best they could!

**ALSO** this PR should fix how currently live adventures, please try to find public adventures and test them locally. Here's some:
- https://hedy.org/public-adventures?level=5&lang=&tag=&search=lootjes
- https://hedy.org/public-adventures?level=5&lang=&tag=&search=Schildpad
- https://hedy.org/public-adventures?level=13&lang=&tag=&search=bonus
